app/controllers/application_controller.rb

```diff
@@ -1026,14 +1025,14 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
 
       custom = MiqReport.all.sort_by { |r| [r.rpt_type, r.filename.to_s, r.name] }
       rep = custom.select do |r|
-        r.rpt_type == "Custom" && (user.admin_user? || r.miq_group_id.to_i == session[:group].to_i)
+        r.rpt_type == "Custom" && (user.admin_user? || r.miq_group_id.to_i == current_group.try(:id))
```

Comment:
I went back and forth on this one.

`group == current_group`, for 3 of the 4 method calls.

From `build_report_listnav` it is possible, that the `group` is from `role_choice`. In that case, I think `current_group` is not the same as `group`. But this dates back to pre 2011.

We'll probably need to verify this for tenancy and reporting